Unity-Technologies/com.unity.multiplayer.samples.bitesize
# Renamed "Api diorama" into "Multiplayer Use Cases" (#MTT-6885)

### Description

- Renamed "Api diorama" into "Multiplayer Use Cases"

### Issue Number(s)
- [#MTT-6885](https://jira.unity3d.com/browse/MTT-6885)

### Contribution checklist

### How to validate

Ignore the changes to the files, as renaming causes all files to be marked as changed.
Do a full text search for "Diorama" (case insensitive) and check that there are 0 references.
